src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -131,6 +131,18 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Critical,
                 formatString: "{failure}");
 
+        private static readonly Action<ILogger, Exception> _excessivePermissions =
+            LoggerMessage.Define(
+                eventId: new EventId(21, "ExcessivePermissions"),
+                logLevel: LogLevel.Warning,
+                formatString: "The process was launched at an administrative level and will have access to all processes on the system. The dotnet-monitor process should be launched as the same user as the target dotnet process.");
+
+        private static readonly Action<ILogger, Exception> _ntlmDisabled =
+            LoggerMessage.Define(
+                eventId: new EventId(22, "ExcessivePermissions"),
```

Comment:
This event name needs to be unique. I would suggest "DisabledNegotiateWhileElevated".

Reply:
And rename the field and method to correspond.